google/cloud/bigquery/client.py

```diff
@@ -1545,12 +1649,18 @@ def delete_table(
                 path=path,
                 timeout=timeout,
             )
-        except google.api_core.exceptions.NotFound:
+        except core_exceptions.NotFound:
             if not not_found_ok:
                 raise
 
     def _get_query_results(
-        self, job_id, retry, project=None, timeout_ms=None, location=None, timeout=None,
+        self,
+        job_id: str,
+        retry: retries.Retry,
+        project: str = None,
+        timeout_ms: int = None,
+        location: str = None,
+        timeout: float = None,
     ):
```

Comment:
Any reason we don't have a return type here?

```suggestion
    ) -> _QueryResults:
```